### ISSUE TITLE
Minicpm-V2.6 LoRA Fine-tuning.

### ISSUE THREAD

**Issue by 2U1** [OpenBMB/MiniCPM-V#489]
Thanks for your great work!
I'm a bit curious about when fine-tuning the model with LoRA. It seems like when I set the option `tune_vision=true` and `tune_llm=false`, the vision model and both language model adapter is tuned. Howevere when the looking into the output directory, there is only the adapter weight. 
Is the weight of the vision model saved into the adapter weight too? or should I save the vision model weight with it?

Also, is it better to set a different learning rate for vision and language model when using LoRA?

If so, I'll make a PR for it.
